package.json

```diff
@@ -33,7 +35,7 @@
                     "Protocol Buffer Text Format",
                     "Textual Protobuf",
                     "pbtxt",
-                    "textproto"
+                    "prototxt"
```

Comment:
Would also like to keep the `textproto` language in edition to `prototxt`. A search on GitHub reveals a very large number of `textproto` files as well.

Reply:
Restored in  50b89f3.